source/posix/time.c

```diff
@@ -54,17 +54,17 @@
  */
 #    include <time64.h>
 
-static const time_t s_time_max = ~(1L << (sizeof(time_t) * CHAR_BIT - 1));
-static const time_t s_time_min = (1L << (sizeof(time_t)) * CHAR_BIT - 1));
+static const time_t s_time_max = ~(1L << ((sizeof(time_t) * __CHAR_BIT__ - 1)));
```

Comment:
Without proper header, `CHAR_BIT` is undefined. `__CHAR_BIT__` is defined.

Reply:
operator `<<` and `-` have different priority, compiler will give warning about `<<` has lower priority than `-`, even though this is exactly what we want.